README.md

```diff
@@ -182,6 +182,7 @@ will be thrown by `minimalcss`. If you know it's safe to ignore errors (for exam
 third-party webpages), set this to `true`.
 * `styletags` - If set to `true`, on-page `<style>` tags are parsed along with external stylesheets. By default, only external stylesheets are parsed.
 * `enableServiceWorkers` - By default all Service Workers are disabled. This option enables them as is.
+* `isJavaScriptEnabled` - By default JavaScript is enabled. This option allows disable JavaScript.
```

Comment:
This name isn't right. Look at the option above for example. "isJavascriptEnabled" is *state* not a command. 

Also, this option needs to either explain how it coincides with the (admittedly poorly named) `withoutjavascript` option, or mention this option where `withoutjavascript` is documented.